### DISCUSSION TITLE
What is the difference with "azure-search-openai-demo" repo?

### DISCUSSION THREAD

**Discussion by ssainz** [microsoft/sample-app-aoai-chatGPT#257] [Q&A]
Hello, 
Noticed that both: 

https://github.com/microsoft/sample-app-aoai-chatGPT
and 
https://github.com/Azure-Samples/azure-search-openai-demo/

Are kind of similar, are both from MicroSoft and if so, what are the differences?

**Comment by pamelafox** [comment]
I have an FAQ about that in the other repo:
https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/README.md#faq

I need to update that now to include the fact that this Repo now includes conversation history via CosmosDB.

**Reply by pamelafox** [reply]
More detail in https://github.com/Azure-Samples/azure-search-openai-demo/pull/640/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R318
